### PR TITLE
Add hypgen params to auth redirect

### PR DESCRIPTION
**Pull request by bryce-fitzsimons**
### Features and Changes

- Closes **(add link to issue here)**

### CODE REVIEW

File: packages/back-end/src/services/auth/OpenIdAuthConnection.ts [hunk 1]
```diff
@@ -37,6 +37,8 @@ if (USE_PROXY) {
   custom.setHttpOptionsDefaults(getHttpOptions());
 }
 
+const passthroughQueryParams = ["hypgen", "hypothesis"];
```
Comment (bryce-fitzsimons) [inline packages/back-end/src/services/auth/OpenIdAuthConnection.ts:40]: `hypgen=1` triggers the new layout
`hypothesis=foo+bar` renders the hypothesis text onto the page

Comment (bttf) [reply packages/back-end/src/services/auth/OpenIdAuthConnection.ts:40]: Is it possible to consolidate into `hypothesis`? (Check if it's non-empty for new layout)

**Review by bttf — COMMENTED** [review]
Like we talked, probably good to have @jdorn take a look

File: packages/back-end/src/services/auth/OpenIdAuthConnection.ts (hunk 1 above)
Comment (bryce-fitzsimons) [reply packages/back-end/src/services/auth/OpenIdAuthConnection.ts:40]: it is, but thought it might be good to keep things separate

Comment (bryce-fitzsimons) [reply packages/back-end/src/services/auth/OpenIdAuthConnection.ts:40]: (also bc we'll eventually import multiple hypotheses/variants at once for bandits, in which case we likely wouldn't render hypothesis text)

Comment (bttf) [reply packages/back-end/src/services/auth/OpenIdAuthConnection.ts:40]: Seems premature but I'll allow it 🤷🏾

**Review by bttf — APPROVED** [review]

**Pull request merged**